thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/api/AnomalyFeedbackApi.java

```diff
@@ -24,6 +27,10 @@ public class AnomalyFeedbackApi {
   private UserApi owner;
   private AnomalyFeedbackType type;
   private String comment;
+  private Date created;
+  private Date updated;
+  private String updatedBy;
```

Comment:
it's not clear in the ticket but let's use the `UserApi owner` instead of introducing a `updatedBy`. 
you can see how it's done in `AlertApi`.

Reply:
ack

Reply:
Can we expose `createdBy` and `updatedBy`

Reply:
Want to ask another clarification question, if we use `owner` instead, does it mean whoever update the feedback would become the owner of this feedback object? In DTO, we still maintain the `createdBy` and `updatedBy` and `createdBy` would be the user who submitted the original feedback and `updatedBy` would be someone who makes a change.